packages/components/src/Tag/Tag.spec.tsx

```diff
@@ -0,0 +1,8 @@
+// @flow
```

Comment:
We need to remove the flow stuff when moving these files over

Reply:
ah good catch, I assumed we were running `flow-to-ts` on these which would remove them. One of our cleanup tasks for this period will be writing a script to do all this